api/resources/views/job-form/edit.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        @include('errors')
+        <form action="{{ route('job-forms.update', $jobForm->id) }}" method="post">
+            @csrf
+            @method('put')
+
+            <div class="form-group">
+                <label for="name">Form Name</label>
+                <input type="text" class="form-control form-control" id="name" name="name" placeholder="Enter form name" value="{{$jobForm->name}}">
+            </div>
+
+            @foreach($fields as $field)
+                <div class="form-group">
+                    <label style="text-transform: capitalize" for="select-{{$field}}">{{ implode(" ", explode('_', $field)) }} field</label>
+                    <select class="custom-select custom-select-lg" name="{{$field}}">
+                        <option disabled>Select status for {{ implode(" ", explode('_', $field)) }} field</option>
```

Comment:
I'd prefer there was a default choice, either unused or optional, instead of having to manually choose everything every time. Plus, fields reset after failed submit here too.

Reply:
No default choice and will fix that reset

Reply:
It is an edit so, it has predifend values, no reset to fix in here